FileIO/VtkIO/VtuInterface-impl.h

```diff
@@ -0,0 +1,79 @@
+/**
+ * \file VtuInterface-impl.h
+ * \author Lars Bilke
+ * \date   2014-09-25
+ * \brief  Implementation of the VtuInterface class.
+ *
+ * \author Wenqing Wang
+ * \date   2015-10-19
+ * \brief  Added parallel output of vtu pieces, and visualisation
+ *         of the vtu picess via pvtu.
+ *
+ * \copyright
+ * Copyright (c) 2012-2015, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ */
+
+#include "VtuInterface.h"
+
+#include <vtkNew.h>
+#include <vtkSmartPointer.h>
+#include <vtkUnstructuredGrid.h>
+
+#include <logog/include/logog.hpp>
+
+#include "InSituLib/VtkMappedMeshSource.h"
+
+namespace FileIO
+{
+
+template<typename UnstructuredGridWriter>
+bool VtuInterface::writeVTU(std::string const &file_name, const int num_partitions)
+{
+	if(!_mesh)
+	{
+		ERR("VtuInterface::write(): No mesh specified.");
+		return false;
+	}
+
+	// See http://www.paraview.org/Bug/view.php?id=13382
+	if(_data_mode == vtkXMLWriter::Appended)
+		WARN("Appended data mode is currently not supported, written file is not valid!");
```

Comment:
Either make it an error and abort here, or change the _data_mode to something which will produce valid results; the latter needs to be indicated in the warning message.